docs/build.md

```diff
@@ -269,6 +269,57 @@ cmake --build build/mac --config Release --target vulkan_samples -j4
 ./build/mac/app/bin/Release/x86_64/vulkan_samples --help
 ```
 
+## iOS
+
+### Dependencies
+
+- CMake v3.19+
+- XCode v12 for Apple Silicon
+- Command Line Tools (CLT) for Xcode `xcode-select --install`
+- [Vulkan SDK](https://vulkan.lunarg.com/doc/sdk/latest/mac/getting_started.html) `./install_vulkan.py`
+- [CMake Options](#cmake-options)
+- [3D models](#3d-models)
+
+### Build with CMake
+
+`Step 1.` The following command will generate the project
+
+```
+cmake -H. -Bbuild/ios -DCMAKE_BUILD_TYPE=Release -G Xcode -DCMAKE_SYSTEM_NAME=iOS -DCMAKE_OSX_DEPLOYMENT_TARGET=11.0
+```
+
+`Step 2.` Build the project
+
+```
+cmake --build build/ios --config Release --target 
+vulkan_samples -j4 -sdk iphonesimulator -allowProvisioningUpdates
+```
+
+NB: The above creates a bundle.  It is possible to build 
```

Comment:
Is it worth adding Xcode instructions?

For me this was the following with my iphone plugged in

On the Iphone
1. Go the Settings app, and navigate to Privacy & Security > Developer Mode
2. Restart when prompted

On the macbook
1. Open Xcode -> Open the Xcode project from build/ios
3. Set the target to vulkan_samples
4. Follow code signing steps https://developer.apple.com/documentation/xcode/running-your-app-in-simulator-or-on-a-device
5. Set the device to a simulator (default) or the plugged in iphone
6. Click Play

We could also make an issue to follow up with these docs and do it at a later date as these would likely require screenshot examples

Reply:
There are a lot of code signing steps but I eventually got the application to run

Reply:
I have now gotten this to work without XCode needing to be open.  However, now, you can simply open the project in XCode and hitting the play button; it'll simply work no need for the extra work.